sklearn/impute/_iterative.py

```diff
@@ -581,6 +582,23 @@ def fit_transform(self, X, y=None):
             self.n_iter_ = 0
             return super()._concatenate_indicator(Xt, X_indicator)
 
+        # Convert min and max values to float arrays if array-like
+        # Else (i.e, if scalar) broadcast it to an array of shape=n_features
+        self._min_value = \
+            as_float_array(self.min_value, force_all_finite=False) \
```

Comment:
Use parenthesised expression rather than `\` for line continuation, please.

Reply:
Done!